command_parse.h

```diff
@@ -65,6 +65,9 @@ enum CommandResult parse_unsubscribe     (struct Buffer *buf, struct Buffer *s,
 enum CommandResult parse_unsubscribe_from(struct Buffer *buf, struct Buffer *s, intptr_t data, struct Buffer *err);
 #endif
 
+enum CommandResult mutt_parse_rc_line_cwd(const char *line, struct Buffer *err, char *cwd);
+char *mutt_get_sourced_cwd(void);
```

Comment:
is (void) necessary? I think it flags argument-less function rather than unknown argd function?

Reply:
Yes.  in C++ `function()` specifically means no parameters.
In C, it means unknown, which means the compiler can't issue warnings.